tox.ini

```diff
@@ -25,10 +25,10 @@ commands = flake8
 
 # Set flake8 configuration options which are used by the `flake8` command in [testenv:pep8]
 [flake8]
-# E501 ignore line longer than 80
```

Comment:
Okay... so now the limit is 140 (the pep8 default?) right?

Reply:
Yes, its 140, set below: `max-line-length = 140`